terraform/cloud-platform-components/prometheus.tf

```diff
@@ -0,0 +1,74 @@
+resource "helm_repository" "coreos" {
+  name = "coreos"
+  url  = "https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/"
+}
+
+resource "kubernetes_storage_class" "prometheus" {
+  metadata {
+    name = "prometheus-storage"
+  }
+
+  storage_provisioner = "kubernetes.io/aws-ebs"
+
+  parameters {
+    type = "gp2"
+  }
+}
+
+resource "helm_release" "prometheus_operator" {
+  name          = "prometheus-operator"
+  chart         = "coreos/prometheus-operator"
+  namespace     = "monitoring-terraform"
+  recreate_pods = "true"
+
+  depends_on = [
+    "null_resource.deploy",
+    "helm_repository.coreos",
+  ]
+}
+
+resource "helm_release" "kube_prometheus" {
+  name          = "kube-prometheus"
+  chart         = "coreos/kube-prometheus"
+  namespace     = "monitoring-terraform"
```

Comment:
Same as above.

Reply:
Yeah, good catch. Will get a commit in now.

Reply:
ce51bd4